railties/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+*   Set `skip_dev_gems` to be a `class_option` so it can be invoked from the
```

Comment:
```suggestion
*   Add `--skip-dev-gems` option to app generator to skip adding development gems (like `web-console`) to the Gemfile
```

I think the changelog entry should be more focused on what has changed for users vs how the implementation has changed

Reply:
Changelog updated as suggested.